grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ReasoningTests.java

```diff
@@ -29,9 +29,11 @@
 
 
 import static ai.grakn.test.GraknTestEnv.usingTinker;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.not;
 import static org.junit.Assume.assumeTrue;
 
-public class Tests{
+public class ReasoningTests {
```

Comment:
Then please detail this in java docs otherwise there is a risk of the test classes being merged.